corehq/apps/hqcase/api/get_list.py

```diff
@@ -65,6 +65,7 @@ def _to_date_filters(field, date_filter):
     'owner_id': case_es.owner,
     'case_name': case_es.case_name,
     'closed': lambda val: case_es.is_closed(_to_boolean(val)),
+    'index': case_search.reverse_index_case_query,
```

Comment:
This is great. I'm not sure if `index` is the right name though. Would `parent` not be more clear / better understood? Perhaps that's also not quite right since this would return all indexed cases, both child and extension cases. Maybe `related_to`? Perhaps the naming should be put to a wider audience to decide.

Reply:
I'd support "related_to".

Reply:
I like `related_to`, though it's notable that this only travels one direction (and only one step), so it would only find parents, not children or grandparents, which are arguably also "related".

Before I found `reverse_index_case_query`, I wrote a similar filter called `has_index_to`, which I thought was a pretty accurate way to describe what this does, though that doesn't feel like the right phrasing for an API.  Couple other ideas I thought of:

- `indexes`, as in "this case indexes X", but that sounds too much like "indices"
- `index.<identifier>`, though I'm not crazy about that, as I don't think that degree of specificity is needed, and it feels like additional friction

I'll ask around a little for opinions.

Reply:
> index.<identifier>, though I'm not crazy about that, as I don't think that degree of specificity is needed, and it feels like additional friction

I think the specificity might be useful and was tempted to ask about it originally but didn't since by not specifying it we return all indexed cases which can always be filtered on the client side.

I agree that these other names are not ideal either. Interested to see what others suggest.